lib/help.js

```diff
@@ -39,4 +40,5 @@ module.exports = `
       $ tb --list pending coding
       $ tb --archive
       $ tb --restore 4
+      $ tb --rename-board My_Board Todo
```

Comment:
Nitpick: But can we keep the stylizing of the board names consistent. Elsewhere, boards are referred to as all lower case, single words. Here, you've introduced a camel-case, "_" separated pattern that clashes a bit with the above examples

Reply:
Why not use quotes?
This would be the only place in the application where we need to replace spaces with underscores.

Why not:
``tb --rename-board "My Board" Todo``
?